NOLF2/ClientShellDLL/TO2/ScreenProfile.cpp

```diff
@@ -139,7 +139,8 @@ LTBOOL CScreenProfile::Build()
 	AddControl(m_pDlg);
 
 	m_pDlg->SetBasePos(dlgPos);
-	m_pDlg->SetScale(g_pInterfaceResMgr->GetXRatio());
+	m_pDlg->ApplyPosition(g_pInterfaceResMgr->GetYRatio(), g_pInterfaceResMgr->Get4x3Offset());
+	m_pDlg->SetScale(g_pInterfaceResMgr->GetYRatio());
```

Comment:
CLTGUIWindow::SetScale was renamed to ApplyPosition and it calls CLTGUICtrl::SetScale already